### PR TITLE
Created input that takes in company name

### PR DESCRIPTION
**Pull request by DHua5922**
* **Please check if the PR fulfills these requirements**
- [X] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)

New Feature: input that takes in company name.

* **What is the current behavior?What does the code do before you made the new changes?** 

page for user input is empty

* **What is the new behavior?**

page for user input now has an input that takes in a company name.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)

No

* **Other information**:

### CODE REVIEW

File: frontend/src/components/CompanyInput/CompanyInput.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    Input, 
+    InputGroup, 
+    InputGroupAddon, 
+    Button 
+} from "reactstrap";
+import "./CompanyInput.css";
+
+export default function CompanyInput() {
+
+    const [companyName, setCompanyName] = useState("");
```
Comment (Codubee) [inline frontend/src/components/CompanyInput/CompanyInput.js:14]: If were going to be using state id rather us create a class component than use the hooks in a functional component.

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Dylan. Just wanted to discuss one thing.

File: frontend/src/components/CompanyInput/CompanyInput.js (hunk 1 above)
Comment (DHua5922) [reply frontend/src/components/CompanyInput/CompanyInput.js:14]: What is wrong with hooks?

**Review by Codubee — APPROVED** [review]
LGTM

**Comment by sunnyguan** [comment]
Looks good to me too!

**Review by sunnyguan — APPROVED** [review]

**Pull request merged**